### ISSUE TITLE
Update k8s 1.13, 1.14 and 1.15 versions on Rancher 2.2.x

### ISSUE THREAD

**Issue by alena1108** [rancher/rancher#23043]
Have to:

- [ ] Refresh k8s versions for 1.13, 1.14 and 1.15 lines on release/v2.2, in `rancher/types`
- [ ] Backport - forwardport? - versions to rancher master, `rancher/kontainer-driver-metadata`, for backwards compatibility

**Comment by prachidamle** [comment]
- To take care of future upgrades of v2.2.9 to v2.3, we need to update kontainer-driver-metadata with system-images that are supported by v2.2.9 k8s versions.

- In v2.2.9, for versions v1.14.7-rancher1-1 and v1.15.4-rancher1-1, the "WindowsPodInfraContainer" image that v2.3 has does not exist.

- Hence we will be maintaining v1.14.7-rancher1-1 and v1.15.4-rancher1-1 without "WindowsPodInfraContainer" in kontainer-driver-metadata so that upon upgrade the existing clusters on v2.2.9 do not go to updating state.

- In kontainer-driver-metadata v1.14.7-rancher1-2 and v1.15.4-rancher1-2 will be having the windows-infra-pod image that will apply for v2.3 onwards.

**Comment by jiaqiluo** [comment]
The bug fix is validated in Rancher:v2.2-head 9aea30e5

As a standard user, I add three EC2 clusters which have the following k8s versions
```
- 1.15.4 - default 
- 1.14.7
- 1.13.11
```
and each cluster has 
```
- 3 etcd
- 1 control plane
- 3 worker nodes. 
```

All automation tests are passed on those three clusters.
